### ISSUE TITLE
list index out of range error

### ISSUE THREAD

**Issue by golden75** [ablab/rnaquast#3]
Hi I am using rnaQuast 1.5.2 version to run

`rnaQUAST.py \
	--threads 8 \
	--transcripts trinity_K32.Trinity.fasta \
	--reference trinity_combine.fasta.transdecoder.cds \
	--gtf trinity_combine.fasta.transdecoder.gff3 \
	-o output`

Hence it is give me this error:

```
Using non strand specific transcripts...

list index out of range
Traceback (most recent call last):
  File "/isg/shared/apps/rnaQUAST/1.5.2/rnaQUAST.py", line 348, in <module>
    return_code = main_utils()
  File "/isg/shared/apps/rnaQUAST/1.5.2/rnaQUAST.py", line 136, in main_utils
    args.gtf = UtilsAnnotations.clear_gtf_by_reference_chr(args.gtf, ids_chrs, tmp_dir, label_db, logger)
  File "/isg/shared/apps/rnaQUAST/1.5.2/general/UtilsAnnotations.py", line 315, in clear_gtf_by_reference_chr
    gtf_chr = line.strip().split()[0]
IndexError: list index out of range

ERROR! Exception caught!

```

is there any possible explanation for the error?

**Comment by andrewprzh** [comment]
Hi, sorry for a delay. 

This was a known issue in rnaQUAST and should be fixed since version 1.6. Could you give it a try?
v.2.0 can be found here https://github.com/ablab/rnaquast/releases/tag/v.2.0.0 
or installed via conda

**Comment by golden75** [comment]
> Hi, sorry for a delay.
> 
> This was a known issue in rnaQUAST and should be fixed since version 1.6. Could you give it a try?
> v.2.0 can be found here https://github.com/ablab/rnaquast/releases/tag/v.2.0.0
> or installed via conda

Thank you very much for the reply. I will have a go at the newer version and will let you know.
